stream_alert/shared/config.py

```diff
@@ -134,6 +181,25 @@ def load_config(conf_dir='conf/', exclude=None, include=None, validate=True):
 
     return config
 
+def _load_schemas(schemas_dir, schema_files):
+    """Helper to load all schemas from the schemas directory into one ordered dictionary.
+
+    Args:
+        conf_dir (str):  The relative path of the configuration directory
+        schemas_dir (bool): The realtive path of the schemas directory
+
+    Returns:
+        OrderedDict: The sorted schema dictionary.
+    """
+    schemas = dict()
+    for schema in schema_files:
+        schemas_from_file = _load_json_file(os.path.join(schemas_dir, schema), True)
+        dup_schema = set(schemas).intersection(schemas_from_file)
+        if dup_schema:
+            LOGGER.warning('Duplicate schema detected %s. This may result in undefined behavior.',
+                           format(dup_schema))
```

Comment:
what does `format` do here that wouldn't otherwise he handled by the log formatter?

suggestion:
```
LOGGER.warning('Duplicate schema(s) detected: %s. This may result in undefined behavior.',
               ', '.join(sorted(dup_schema)))
```

Reply:
That is actually what I meant, but I put format instead. Whoops.